src/dlls/dbgshim/dbgshim.cpp

```diff
@@ -62,6 +62,90 @@ if it exists, it will:
     } \
     CONTRACTL_END; \
 
+
+//-----------------------------------------------------------------------------
+// Public API.
+//
+// CreateProcessForLaunch - a stripped down version of the Windows CreateProcess
+// that can be supported cross-platform.
+//
```

Comment:
What sort of error handling is expected? I'm guessing right now its just success = TRUE, failure = FALSE and no further info is provided?